File: Mathematics/fibonacci/fibonacci.go
package main

import (
	"fmt"
)

func fibonacci(limit int) {
	if limit >= 1 {
		fmt.Printf("%d", 1)
	}

	a, b := 1, 1

	for b <= limit {
		fmt.Printf(", %d", b)
		a, b = b, a+b
	}

	fmt.Println()
}

func main() {
	fibonacci(8)
	fibonacci(123)
	fibonacci(1)
	fibonacci(50)
	fibonacci(100)
}

File: Mathematics/fibonacci/Fibonacci.java

//Java program to evaluate fibonacci series 

import java.util.Scanner;

public class Fibonacci{
	public static void main(String args[]){
		Scanner inp = new Scanner(System.in);
		int term = inp.nextInt();
		int first =1, second = 1, temp_first=1;
		while(first<=term){
			System.out.print(first+" ");
			temp_first = first;
			first = second;
			second += temp_first;
			
		}
	}
}